### ISSUE TITLE
Improve handling big files/input by ladding layer without loading/displaying it

### ISSUE THREAD

**Issue by darrepac** [qgis/QGIS#54752]
### Feature description

Hello,

Right now (QGIS 3.32), if I want to add a layer from the explorer tree, I have a "add layer to the project" with right click:
![image](https://github.com/qgis/QGIS/assets/7128994/e7f61563-8080-46ae-a336-0f2e5ee29bfa)
It puts the layer into the project and make it visible. When you have very big files or big postgresql database, it could make a lot of time to load. It even seems to preload points in background while not displaying anything yet.
So it takes a lot of time and often we need then to adapt the symbology or other parameters, so a lot of time is lost to display something that is not "right".

I propose to add a new command: "Add layer to the project but not visible". The goal would be to put the layer in the project but not loading it. So it permits to play with all the properties of the layer (symbology etc) before to load anything and waste time for nothing. Then we can make it visible.

It would ease and speed up the handling of big layers.

**Comment by DelazJ** [comment]
> I propose to add a new command: "Add layer to the project but not visible". The goal would be to put the layer in the project but not loading it. So it permits to play with all the properties of the layer (symbology etc) before to load anything and waste time for nothing. Then we can make it visible.

@darrepac there is already an option to add layers without automatic rendering (see https://docs.qgis.org/3.28/fr/docs/user_manual/introduction/qgis_configuration.html#rendering-settings)

**Comment by darrepac** [comment]
Indeed, wouldn't the experience be better by adding a drop menu item?
